app/assets/stylesheets/media.scss

```diff
@@ -119,15 +119,19 @@
     }
 
     #sul-embed-iframe-code {
-      width: 100%;
+      width: calc(95% - 5%);
```

Comment:
I meant like `calc(100% - 2rem)`.  There's no need to calc two percentages as we could do that ahead of time.